<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1241</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(16, 48, 23);
font: 24pt &quot;Helvetica&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="USALabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(176, 173, 177);</string>
    </property>
    <property name="text">
     <string>United States</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="UKLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(176, 173, 177);</string>
    </property>
    <property name="text">
     <string>United Kingdom</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sovietLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>340</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(176, 173, 177);</string>
    </property>
    <property name="text">
     <string>Soviet Union</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="unitedStatesFlag">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="unitedKingdomFlag">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sovietFlag">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="japanFlag">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>180</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="germanLabel">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>40</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(176, 173, 177);</string>
    </property>
    <property name="text">
     <string>Germany</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="italyFlag">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>330</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="germanFlag">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>30</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="italyLabel">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>340</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(176, 173, 177);</string>
    </property>
    <property name="text">
     <string>Italy</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="japanLabel">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>190</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(176, 173, 177);</string>
    </property>
    <property name="text">
     <string>Japan</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="USAScore">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 161, 159);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="UKScore">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>260</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 161, 159);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sovietScore">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>410</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 161, 159);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="japanScore">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>250</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 161, 159);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="germanScore">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>100</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 161, 159);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="italyScore">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>400</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 161, 159);</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>530</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 25);
border-radius: 12%;</string>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>530</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(128, 255, 155);
border-radius: 12%;</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="battleLog">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>500</y>
      <width>451</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;the dead saloon&quot;;
color: rgb(208, 205, 210);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="allyLabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>370</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(176, 173, 177);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="axisLabel">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>370</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(176, 173, 177);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="vsLabel">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>440</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>vs</string>
    </property>
   </widget>
   <widget class="QLabel" name="USADefeat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>341</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="UKDefeat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>351</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="sovietDefeat">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>341</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="germanyDefeat">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>20</y>
      <width>341</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="japanDefeat">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>180</y>
      <width>341</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="italyDefeat">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>340</y>
      <width>341</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="winLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>481</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 48pt &quot;the dead saloon&quot;;
color: rgb(206, 203, 208);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1241</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
